apps/www/components/ActionBar/Audio/Info.js

```diff
@@ -0,0 +1,221 @@
+import useUnsubscribeDocumentMutation from '../graphql/useUnsubscribeDocumentMutation'
+
+const styles = {
+  audioInfo: css({
+    margin: 0,
+    textAlign: 'left',
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    whiteSpace: 'pre-wrap',
+    ...convertStyleToRem(fontStyles.sansSerifRegular14),
+    [mediaQueries.mUp]: {
+      ...convertStyleToRem(fontStyles.sansSerifRegular15),
+    },
+  }),
+}
+
+const Contributors = ({ contributors }) => {
+  const { t } = useTranslation()
+
+  const names = intersperse(
+    contributors.map((s) =>
+      s.user?.slug ? (
+        <Link href={`/~${s.user.slug}`} passHref>
+          <Editorial.A>{s?.user?.name || s.name}</Editorial.A>
+        </Link>
+      ) : (
+        s.name
+      ),
+    ),
+    (_, i) => <span key={i}>, </span>,
+  )
+
+  return t.pluralize.elements('article/actionbar/audio/info/voices', {
+    names,
+    count: contributors.length,
+  })
+}
+
+const PlaySyntheticReadAloud = ({ onPlay, children }) => {
+  const { t } = useTranslation()
+  const [colorScheme] = useColorContext()
+
+  return (
+    <span {...colorScheme.set('color', 'textSoft')}>
+      <button
+        {...plainButtonRule}
+        style={{ textDecoration: 'underline', display: 'inline' }}
+        onClick={onPlay}
+      >
+        {children ? children : t('article/actionbar/audio/info/play-synth')}
+      </button>
+    </span>
+  )
+}
+
+const SubscribeReadAloud = ({ document }) => {
+  const [colorScheme] = useColorContext()
+  const { t } = useTranslation()
+
+  const { data, loading: loadingQuery } = useDocumentSubscriptionQuery({
+    variables: {
+      path: document.meta?.path,
+      onlyMe: true,
+    },
+    skip: !document.meta?.path,
+  })
+
+  const [subscribe, { loading: loadingSubscribe }] =
+    useSubscribeDocumentMutation()
+  const [unsubscribe, { loading: loadingUnsubscribe }] =
+    useUnsubscribeDocumentMutation()
+
+  const loading = loadingQuery || loadingSubscribe || loadingUnsubscribe
+
+  const readAloudSubscription = data?.document?.subscriptions?.nodes.find(
+    ({ object: { id } }) => id === document.id,
+  )
+
+  const isSubscribed =
+    readAloudSubscription?.filters.includes(EventObjectType.READ_ALOUD) &&
+    readAloudSubscription?.active &&
+    readAloudSubscription?.isEligibleForNotifications
+
+  const handleMutation = useMemo(
+    () => async () => {
+      try {
+        if (isSubscribed) {
+          await unsubscribe({
+            variables: {
+              subscriptionId: readAloudSubscription.id,
+              filters: [EventObjectType.READ_ALOUD],
+            },
+          })
+        } else {
+          await subscribe({
+            variables: {
+              documentId: document.id,
+            },
+          })
+        }
+      } catch (e) {
+        console.error(e)
+      }
+    },
+    [readAloudSubscription],
+  )
+
+  if (!readAloudSubscription?.isEligibleForNotifications) {
```

Comment:
@patrickvenetz This here is false for me when developing against staging.
If this is false, the callout renders without content. Perhaps if this is false, we should not render the option to subscribe at all and instead fallback to play synthetic if that is available.